cookbooks/cdo-users/templates/default/ssh_config.erb

```diff
@@ -1,4 +1,4 @@
-Host staging test levelbuilder-* production-daemon production-console i18n-dev adhoc-* *.ec2.internal *.cdn-code.org
+Host staging test levelbuilder production-daemon production-console i18n-dev adhoc-* *.ec2.internal *.cdn-code.org
```

Comment:
Verified that by applying this change to my local `~/.ssh/known_hosts` file on gateway, I am successfully able to connect to the new server with `ssh levelbuilder`